composer.json

```diff
@@ -8,7 +8,7 @@
     },
     "require-dev": {
         "ext-xdebug": "*",
-        "phpunit/phpunit": "^5.4",
+        "phpunit/phpunit": "^5.4|^6.5",
```

Comment:
Was there a feature that needed 6.5?

If there are reasons please raise an issue and then link the issue in the commit or PR

Reply:
According to the PHPUnit [website](https://phpunit.de/), the ```php-7.2+``` needs the PHPUnit ``6+```.

Reply:
Interesting, thanks